example.config.toml

```diff
@@ -18,11 +18,17 @@ reconnect = false
 
 [parameters]
 url_limit = 10
-accept_lang = "en"
 status_channels = []
 nick_response_str = ""
 reconnect_timeout = 10
 
+[http]
+timeout_s = 10
+max_redirections = 10
+max_retries = 3
+retry_delay_s = 5
+accept_lang = "en"
```

Comment:
Looks like this is missing `user_agent`?

Reply:
Yep! `user_agent` is now `Option<String>`, so the default is `None`, and therefore it's not serialised to TOML.

Since now, if the user agent isn't configured, a user agent is generated automatically (including the bot version), there needs to be some way to tell if it's been set in the configuration or not, and `Option<_>` seemed much cleaner than looking for an empty string. It is documented in the readme, though.